xml/html_title.xml

```diff
@@ -3606,12 +3606,11 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:ntop:ntopng:-"/>
   </fingerprint>
 
-   <fingerprint pattern="^FreeNAS$">
-     <description>FreeNAS</description>
-     <example>FreeNAS</example>
-     <param pos="0" name="service.vendor" value="iXsystems"/>
-     <param pos="0" name="service.product" value="FreeNAS"/>
-     <param pos="0" name="service.cpe23" value="cpe:/a:ixsystems:freenas:-"/>
+  <fingerprint pattern="^FreeNAS$">
+    <description>FreeNAS</description>
```

Comment:
A couple of notes that I probably should have made on PR #382 and which aren't required for this PR

- We may wish to expand the `description` field to be `iXsystems FreeNAS`
- We can assert `os` params here though not `hw` because it's open source and can be deployed on custom gear
```
h:ixsystems:freenas
o:ixsystems:freenas_firmware
```
- `os.family` should be `<param pos="0" name="os.family" value="FreeBSD"/>`